lib/bgs/service.rb

```diff
@@ -196,6 +196,13 @@ def get_regional_office_by_zip_code(zip_code, country, province, lob, ssn)
       '347' # return default location id
     end
 
+    def find_regional_offices
+      find_regional_offices_response = service.data.find_regional_offices
+      find_regional_offices_response[:return]
```

Comment:
```suggestion
      find_regional_offices_response = service.data.find_regional_offices[:return]
```

Would this work?

Reply:
Yes!  Thank you - I've updated as suggested.